File: solutions/r/pangram/1/pangram.R
is_pangram <- function(input) {
  # Convert the input string to lowercase
  input_lower <- tolower(input)
  
  # Split the string into individual characters
  chars <- strsplit(input_lower, "")[[1]]
  
  # Check if all 26 letters of the English alphabet are present in the characters
  all(letters %in% chars)
}

File: solutions/vbnet/pangram/1/Pangram.vb
Imports System.Linq

Public Module Pangram
    Const alphabet As String = "abcdefghijklmnopqrstuvwxyz"

    Public Function IsPangram(ByVal input As String) As Boolean
        Return alphabet.All(AddressOf input.ToLower().Contains)
    End Function
End Module